### PR TITLE
feat: code action for adding `@moduledoc false`

### LINKED ISSUE #11: Code Action: Credo.Check.Readability.ModuleDoc - add `@moduledoc false`
## Description

Add a code action to inject `moduledoc @false` to fix the `Credo.Check.Readability.ModuleDoc` check.

### PR DESCRIPTION
**Pull request by wesleimp**
image

Closes #11

### CODE REVIEW

File: lib/credo_language_server/code_actions/module_doc.ex [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    [
+      %GenLSP.Structures.CodeAction{
+        title: "Add \"@moduledoc false\"",
+        edit: %GenLSP.Structures.WorkspaceEdit{
+          changes: %{
+            uri => [
+              %GenLSP.Structures.TextEdit{
+                new_text: "  @moduledoc false\n",
```
Comment (mhanberg) [inline lib/credo_language_server/code_actions/module_doc.ex:15]: I think this is fine for now, and matches the behavior of the existing action, but ti would be nice if we could make the new text follow the same amount of indent as the surrounding text, that would be nice.

File: lib/credo_language_server/code_actions/module_doc.ex [hunk 2]
```diff
@@ -0,0 +1,24 @@
+defmodule CredoLanguageServer.CodeActions.ModuleDoc do
```
[the remark this answers is not in the document]
Comment (wesleimp) [reply lib/credo_language_server/code_actions/module_doc.ex:1]: I don't like much the idea of nested modules, but for now I think it's ok

File: lib/credo_language_server/code_actions/module_doc.ex (hunk 1 above)
Comment (wesleimp) [reply lib/credo_language_server/code_actions/module_doc.ex:15]: Yeah, I agree. Maybe it breaks with nested modules. I'm gonna try a better approach

**Comment by mhanberg** [comment]
@wesleimp I pushed some updates to this branch and merged it in #23 

I was able to get the correct indenting working, and figured out a foundation for the code actions that I liked.

Thanks for getting it started!

**Pull request closed without merging**